keyboards/wilba_tech/wt65_xt/keymaps/zunger/keymap.c

```diff
@@ -0,0 +1,659 @@
+/* Copyright 2021 Yonatan Zunger
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#include <assert.h>
+#include "wt65_xt.h"
```

Comment:
Isn't needed.  `QMK_KEYBOARD_H` is generated as this, already.
```suggestion
```

Reply:
Done.